tests/testthat/test-expr_string.R

```diff
@@ -685,28 +685,28 @@ test_that("str$str_explode", {
 })
 
 
-test_that("str$parse_int", {
+test_that("str$to_integer", {
   expect_identical(
-    pl$lit(c("110", "101", "010"))$str$parse_int(2)$to_r(),
+    pl$lit(c("110", "101", "010"))$str$to_integer(2)$to_r(),
     c(6, 5, 2)
   )
 
   expect_identical(
-    pl$lit(c("110", "101", "010"))$str$parse_int()$to_r(),
+    pl$lit(c("110", "101", "010"))$str$to_integer()$to_r(),
```

Comment:
isn't this wrong since the default base changed?

Reply:
Yeah, I forgot to update the test.